server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/datatransfer/DataTransferCallable.java

```diff
@@ -18,25 +18,30 @@
 
 import java.util.concurrent.Callable;
 
-import com.oceanbase.odc.plugin.task.api.datatransfer.model.TransferObjectsInfo;
+import com.oceanbase.odc.plugin.task.api.datatransfer.model.DataTransferTaskResult;
 
 /**
  * @author liuyizhuo.lyz
  * @date 2023-09-15
  */
-public interface DataTransferCallable extends Callable<TransferObjectsInfo> {
+public interface DataTransferCallable extends Callable<DataTransferTaskResult> {
 
     /**
      * get current task status for monitoring.
      * 
      * @return transfer result, including data objects and schema objects. The two are independent of
      *         each other. An object can appear in two lists at the same time.
      */
-    TransferObjectsInfo getStatus();
+    DataTransferTaskResult getStatus();
```

Comment:
why does user get a result object when he/she call this method? what's the differences between this method and `call`?

Reply:
removed this method. Replaced it by two methods `getDataObjectsStatus` and `getSchemaObjectsStatus`